### ISSUE TITLE
(change) Provide a way to configure retriever.githubToken in k8s securely

### ISSUE THREAD

**Issue by Yaroslav-Kovalchyk-Imprivata** [thomaspoignant/go-feature-flag#594]
### Motivation

There is no way to define GitHub token securely in case relay-proxy is deployed in k8s via helm chart. 
Below is a sample of the way how githubToken is currently configured. It is required to be defined as simple text, which is not secure.

```
relayproxy:
  config: | # This is a configuration example for the relay-proxy
    listen: 1031
    pollingInterval: 1000
    startWithRetrieverError: false
    retriever:
      kind: github
      repositorySlug: "imprivata-cloud/spring-helm-gofeatureflag-poc"
      path: "goff/feature_control_config.yaml"
      branch: "VT-1906"
      githubToken: "your_secure_token"
    exporter:
      kind: log

image:
  tag: "v1.5.1"
```

### Requirements

Acceptable way to define githubToken is a k8s secret. Please provide such a capability, otherwise relay-proxy with GitHub flags configuration can't be used in real production securely.

**Comment by Yaroslav-Kovalchyk-Imprivata** [comment]
Thank you for providing the env variable name for the token. Sure, I can mount it to a pod, but in this case I can't use your helm chart and have to write my own. Is there any possibility to have such possibility out of the box. And one more thing: is there any doc with all environment variables, that relay proxy supports?

**Comment by thomaspoignant** [comment]
Hey @Yaroslav-Kovalchyk-Imprivata I have updated the helm chart in this PR #597.
It has been released in version `v1.6.0` of GO Feature Flag.

You can see how to use a secret, in the example available in the `values.yaml`:
https://github.com/thomaspoignant/go-feature-flag/blob/30cfa9d3a3c34673fcdf5aed5c04e7ac887ff5da/cmd/relayproxy/helm-charts/relay-proxy/values.yaml#L12-L22

I hope this is what you were looking for.

**Comment by Yaroslav-Kovalchyk-Imprivata** [comment]
Yep. Thanks a lot! ;)
